core/__init__.py

```diff
@@ -2,15 +2,15 @@
 import logging
 import os
 import time
-
+import threading
```

Comment:
From an efficiency perspective, importing the entire `threading` package is too much. Instead, just import the Timer objects from the `threading` package. Make sure to change your object instantiations accordingly.
(`threading.Timer()` -> `Timer()`)

Recommended Changes:
- `from threading import Timer`